.gitignore

```diff
@@ -8,3 +8,6 @@
 /spec/reports/
 /tmp/
 *.csv
+*.pdf
+*.png
```

Comment:
Why do you need to exclude `*.pdf` and `*.png` from this gem? The gem itself is not supposed to generate files. An application using that gem may, and that will be its job to ignore these kinds of files.

As this is a gem, note that the two main ways for someone else to use it should be:

Install and run:
```sh
$ gem install mergometer
$ mergometer <options>
```

Use gem in project:
```rb
$ echo "gem 'mergometer'" >> Gemfile
$ bundle
# Then add code in your project to use mergometer
```

Reply:
But as this gem can also be used as a standalone application, I think we should also ignore pdf and png?

Reply:
No, because when someone uses this gem as a standalone application, then they won't be running it inside the `mergometer` root folder, they shouldn't be in the gem's source.

Reply:
Cool then!